server/modules/surveyRdb/repository/dataView/read.js

```diff
@@ -196,6 +196,25 @@ export const fetchViewData = async (params, client = db) => {
     : client.map(select, queryParams, _dbTransformCallbackSelect({ viewDataNodeDef, nodeDefCols, editMode }))
 }
 
+export const _fetchViewData = async (params) => {
```

Comment:
you can use the fetchViewData already implemented. no need to create a new method